pootle/apps/pootle_store/views.py

```diff
@@ -629,6 +630,26 @@ def handle_suggestion_comment(request, suggestion, comment):
     if comment_form.is_valid():
         comment_form.save()
 
+        if action not in ("accepted", "rejected"):
+            return
+
+        ctx = {
+            'suggestion_id': suggestion.id,
+            'unit_url': request.build_absolute_uri(unit.get_translate_url()),
+            'comment': comment,
+        }
+        if action == "rejected":
+            message = loader.render_to_string(
+                'editor/email/suggestion_rejected_with_comment.txt', ctx)
+            subject = _(u"Suggestion %s rejected with comment" % suggestion.id)            
```

Comment:
I think that in case the suggester receives several emails like this it helps differentiate threads, one for each suggestion.